Loop/View Models/ServicesViewModel.swift

```diff
@@ -6,8 +6,9 @@
 //  Copyright © 2020 LoopKit Authors. All rights reserved.
 //
 
-import LoopKit
 import SwiftUI
+import LoopKit
+import LoopKitUI
```

Comment:
is `LoopKitUI` needed?

Reply:
Yes, the `ServiceDescriptor` type was moved from `LoopKit` to `LoopKitUI`.